admin/schema.php

```diff
@@ -849,8 +851,8 @@
 	user_id					I		NOTNULL DEFAULT '0',
 	name					C(128)	NOTNULL,
 	hash					C(128)	NOTNULL,
-	date_created			I		UNSIGNED NOTNULL DEFAULT '0',
-	date_used				I		UNSIGNED NOTNULL DEFAULT '0'",
+	date_created			I		UNSIGNED NOTNULL DEFAULT '1',
+	date_used				I		UNSIGNED NOTNULL DEFAULT '1'",
```

Comment:
OK this does not really hurt as we alter the columns to set the default to 1 later on, but adding a comment at step 209 explaining the history and reason for the change will help our future selves understand why we're issuing an AlterColumnSQL which does nothing without having to delve through git history.

Reply:
Comment added.